### ISSUE TITLE
Solseek fetch utility adding entry into flatpak cache file

### ISSUE THREAD

**Issue by Vyrnexis** [clintre/solseek#42]
**Describe the bug**
Solseek fetch utility shows 1 installed flakpak, even if no flatpaks are installed. maybe solseek adds an entry into the flatpak cache file for fzf by default if no packages are detected. 

```bash
# If the user has no flatpaks installed, lets create an empty entry for FZF
            if [ ! -s "$FI_CACHE_FILE" ]; then
                touch "$FI_CACHE_FILE"
                echo "Empty|Empty" > "$FI_CACHE_FILE"
                #If it does lets merge the installed entries with available as FP has no command for this
            else
```

**To Reproduce**
open solseek without any flatpaks installed

**Expected behavior**
I would expect that if there are no flatpaks installed then the fetch utility in solseek would display 0

**Screenshots**

<img width="1456" height="472" alt="Image" src="https://github.com/user-attachments/assets/0abe0f01-2c5b-420b-92e4-5101f316db15" />

**System (please complete the following information):**
 - Solus Version: 4.8 (Budgie 10.9.4)
 - Terminal: Kitty
 - Solseek Version: v1.0.0

**Comment by clintre** [comment]
I have fixed this in the 1.0.2 release. It has been pushed to the Solus Unstable repo. Thank you again for finding and reporting this.
